### DISCUSSION TITLE
How do I duplicate my Firefly III Docker instance?

### DISCUSSION THREAD

**Discussion by BugPhobic** [firefly-iii/firefly-iii#6744] [Firefly III] [answered]
I have FireFly III installed through Docker that I am very happy with. I wish to make a duplicate of this instance to run my parents' budget. I basically want the duplicate to have everything that my current install does, except for accounts and transactions.

I was thinking of using the backup for Docker information page from the documentation to this, and then use the backup to accomplish my goal. But is there another way of doing this?

**Comment by JC5** [comment]
That should work just fine :+1:

**Reply by BugPhobic** [reply]
Apologies if this is silly, I've began to learn Docker to use this program.

I was able to successfully follow the backup and restore procedures to migrate my FireFly instance to another system. However, I am can't seem to figure out how to create a SECOND container to run my parents' budgets on.

I tried modifying the docker compose .yml file to use differently named volumes, and then used the docker run command from the install page, but that just overwrote the config I had for my current container.

My goal is to have two containers (with two sets of volumes, I believe? Or can both containers share the same volume?). One container is for my budgets, while the second container is for my parents.

Any idea how I might go about doing that?

**Comment by jamezul** [comment, marked as answer]
Hi BugPhobic,

You have to consider that you're deploying only the FireflyIII portal - you also need a second DB for your parents...
If you're using MySQL or MariaDB as backend DB, i sugest to create a new database (i.e fireflyiii_parents) and use docker-compose file with the new database...you might want to change the container name, host ports and folders mapping(use new folders for your parents instance)

I hope it helps,
